### DISCUSSION TITLE
Include qmd-files for each chapter into a main qmd-file

### DISCUSSION THREAD

**Discussion by MasterGPT9821** [quarto-dev/quarto-cli#8421] [Q&A] [answered]
### Description

I want to include qmd-files for each chapter into a main qmd-file

Code in the main qmd file
```
# include another file

{{ < include _test.qmd > }}
```

Output
![grafik](https://github.com/quarto-dev/quarto-cli/assets/157615015/f99a1800-c6fc-42b6-8bd3-ffc94043a8c9)

Code in the test.qmd
```
## the including quarto file

here is some text
```
![grafik](https://github.com/quarto-dev/quarto-cli/assets/157615015/a46b9198-b332-4fab-aba4-3ae03b629e9d)

**Comment by mcanouil** [comment]
What's the question?

Did you take a look at?

- https://quarto.org/docs/reference/projects/books.html
- https://github.com/quarto-dev/quarto-cli/discussions/7669

Could you share a small self-contained "working" (reproducible) example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

![A placeholder image](https://placehold.co/600x400.png)

The end.
````
`````

**Reply by mcanouil** [reply, marked as answer]
PS: your shortcode is not valid `{{ < include _test.qmd > }}` is incorrect. `{{< include _test.qmd >}}` is.
See <https://quarto.org/docs/authoring/includes.html>.

**Reply by MasterGPT9821** [reply]
You Are right. {{< include test.qmd >}} is correct.
